es-app/src/views/SystemView.cpp

```diff
@@ -172,30 +172,49 @@ void SystemView::onCursorChanged(const CursorState& state)
 	}, (int)(infoStartOpacity * 150));
 
 	unsigned int gameCount = getSelected()->getGameCount();
+	unsigned int favoritesCount = getSelected()->getFavoritesCount();
 
 	// also change the text after we've fully faded out
-	setAnimation(infoFadeOut, 0, [this, gameCount] {
+	setAnimation(infoFadeOut, 0, [this, gameCount, favoritesCount] {
 		std::stringstream ss;
 		
 		// only display a game count if there are at least 2 games
-		if(gameCount > 1)
-			ss << gameCount << " GAMES AVAILABLE";
+	if (gameCount == 1)
+	{
+		ss << gameCount << " GAME AVAILABLE";
+		if (favoritesCount == 1)
```

Comment:
If gameCount is 1 here, how can favoritesCount be greater than that? Seems like "FAVORITES" option can be removed here if counts are being managed correctly.